addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -497,8 +497,8 @@
 		}
 		"424"	//Tomislav
 		{
-			"desp"			"Tomislav: {positive}+40% more accurate, 25% extra spin up time"
-			"attrib"		"106 ; 0.6 ; 87 ; 0.5"
+			"desp"			"Tomislav: {positive}+25% extra switch speed, 25% extra spin up time, {negative}-70% health from healers"
```

Comment:
shouldn't '25% extra spin up time' be '+25% faster spin up time'?

Reply:
in this case, it's applying a 25% faster spin up time on top of an already existing 25% faster spin up time (given as an attribute to all heavy primaries, also explains why it's at 0.5). 

while "extra" was likely added in good spirit to help clear up that it's a bonus, you're right that it would mean slower, but just having `+25% faster spin up time` may confuse players who are reading the description since all miniguns already have it.

maybe combining your suggestion with the current description with something like "Additional/Extra 25% faster spin up time" would clear it up better